routes/publish.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+const { isAuthorized } = require("../middlewares/auth");
+
+import { publish_content } from "../controllers/publish";
+import { checkContentOwnership } from "../middlewares/checkContentOwnership";
+
+// Create router
+const router = express.Router();
+
+// mutations
+router.post(
+  "/:contentId",
+  // isAuthorized,
```

Comment:
This should be commented in, right?
```suggestion
  isAuthorized,
```